### DISCUSSION TITLE
item par default achat classe

### DISCUSSION THREAD

**Discussion by ghosts-arch** [Crownicles/Crownicles#1720] [Nouvelles idées]
proposer un kit d'items standards pour chaque classe (item, bouclier, arme ...) et si on a tout bah on a un bonus (c'est comme la #1699 mais un peu mieux )

ou 

proposer un item par défaut pour chaque classe qui permet quand on change de classe de l'acheter et d'être directement un peu plus op pour la nouvelle classe

je suis pas convaincu de mes idées mais je pose ca quelque part

**Comment by BastLast** [comment]
Possible de modifier lz titre pr un truc qui correspond à la description ?

**Reply by ghosts-arch** [reply]
aucune idée de comment appeller ca

**Reply by BastLast** [reply]
Demande a chat gpt

**Reply by Mw3y** [reply]
"Kit de démarrage complet : bonus garanti" (1) ou "Item de transition pour chaque classe" (2)
